packages/@atjson/document/src/collection.ts

```diff
@@ -2,10 +2,15 @@ import { Annotation, AnnotationJSON, Document, Join } from "./internals";
 
 export function compareAnnotations(a: Annotation<any>, b: Annotation<any>) {
   let startDelta = a.start - b.start;
-  let endDelta = a.end - b.end;
+  let endDelta = b.end - a.end;
```

Comment:
Could you add some comments here about the expected sort order for future visitors?

Some of this is not _super_ intuitive, and I'm sure I'll personally be confused by this in the future 😂

Reply:
//sort by position, then rank, then type :)